crates/store/re_data_loader/src/load_file.rs

```diff
@@ -33,7 +35,7 @@ pub fn load_from_path(
         .into());
     }
 
-    re_log::info!("Loading {path:?}…");
+    re_log::info!("Loading {path:?}...");
 
     // When loading a LeRobot dataset, avoid sending a `SetStoreInfo` message since the LeRobot
     // loader handles this automatically.
```

Comment:
I think it still might be nice conceptually to set the `ApplicationId` to the folder name here. Or is there any other reason to not do this?

Reply:
Oh, sorry GitHub didn't let me reference the lines below, I guess I should have been more clear:

Should we pass the `application_id` for the LeRobot loader here? It would be nice to always have that logic in the same place.

Reply:
Because the LeRobot loader manually sends the `SetStoreInfo` message, currently it doesn't use the settings for that, which I will change 👍